.gitignore

```diff
@@ -58,3 +58,6 @@ specs/**/.ghc.environment.x86_64-linux-*
 
 ## python for generating the doc site's transitive dependencies
 .venv
+
+## Symbolic links
+excludesfile = /home/.gitignore_global
```

Comment:
This should not be here. Ping me if you can't figure it out
```suggestion
```

Reply:
got it